.github/workflows/dapr.yml

```diff
@@ -98,6 +100,14 @@ jobs:
           docker -v
       - name: Parse release version and set REL_VERSION
         run: python ./.github/scripts/get_release_version.py
+      - name: Save release version and release tag
```

Comment:
We can simplify it more by creating the version file inside helmchart artifact directory. then we need one upload step to upload helm chart pkg and version files.

Reply:
simplified